databases/mariadb.libsonnet

```diff
@@ -95,6 +95,7 @@ local p = import 'github.com/jsonnet-libs/kube-prometheus-libsonnet/0.10/main.li
                   + container.withEnvMap({
                     MYSQL_DATABASE: cfg.database,
                     MYSQL_LOWER_CASE_TABLE_NAMES: '1',
+                    MYSQL_DATADIR_ACTION: "upgrade-auto",
```

Comment:
I think it would be good to have this as a property which defaults to `disable`. There might be situations in which automatic upgrades are not prefered

Reply:
updated